projects/batfish/test/org/batfish/main/BatfishTest.java

```diff
@@ -0,0 +1,68 @@
+package org.batfish.main;
+
+import org.batfish.common.BatfishException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class BatfishTest {
+
+    @Test(expected = BatfishException.class)
+    public void testNotExistingPath() {
+        List<Path> result = Batfish.listAllFiles(Paths.get("./test/notExist"));
+    }
+
+
+    @Test
+    public void testNoFileUnderPath() {
+        List<Path> result = Batfish.listAllFiles(Paths.get("./test/noFile"));
+        Assert.assertEquals(0, result.size());
```

Comment:
two things:

1. Usually, functions like `assertEquals` are statically imported so that it doesn't say

    ```java
    Assert.assertEquals(x, y);
    ```

    but rather

    ```java
    assertEquals(x, y);
    ```

    (makes the code slightly easier to read).

2. A better way to do assertions is using [`assertThat`](http://junit.org/junit4/javadoc/4.12/org/junit/Assert.html#assertThat(T, org.hamcrest.Matcher)):

    ```java
    assertThat(results, empty());
    ```

    Using [`empty`](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#empty()). That way when the check fails you get a better error message. Instead of "0 != 3" you get "expected: empty list, got: [a, b, c]". Or something like that.